### ISSUE TITLE
Network Configuration System

### ISSUE THREAD

**Issue by Enzogall** [MathEpitech2025/my_torch#8]
Create a configuration system for network architecture:

* Define configuration file format
* Specify input/hidden/output layers
* Configure activation functions
* Set hyperparameters (learning rate, epochs, etc.)
